src/features/timeline/TimelineElement.tsx

```diff
@@ -0,0 +1,125 @@
+import { max } from 'd3-array';
+import { color as d3color } from 'd3-color';
+import type { ScaleBand, ScaleTime } from 'd3-scale';
+import { scaleOrdinal } from 'd3-scale';
+import { schemeTableau10 } from 'd3-scale-chromatic';
+import type { ForwardedRef } from 'react';
+import { forwardRef } from 'react';
+
+import type { Person } from '@/features/common/entity.model';
+import { eventTypes } from '@/mocks/db';
```

Comment:
in a followup, we could separate these into their own file, to avoid pulling faker into the client bundle when we don't have to. but fine for now since we currently only have mock data anyway.